data.json

```diff
@@ -80,6 +80,15 @@
     "dateAdded": "2018-02-18T07:48:58.000Z",
     "tags": ["clothing", "hacktoberfest", "expired", "stickers"]
   },
+  {
+    "name": "FSFE",
+    "difficulty": "easy",
+    "description": "Fill the form to receive promotional materials from Free Software Foundation Europe!",
+    "reference": "https://fsfe.org/contribute/spreadtheword.en.html",
+    "image": "https://i.imgur.com/TuOjguN.jpg",
```

Comment:
```suggestion
    "image": "https://i.ibb.co/P9XVfPm/FSFE-stickers-medium.jpg",
```

![](https://i.ibb.co/P9XVfPm/FSFE-stickers-medium.jpg)

I'd go for this image instead, it's clearer so we can really know what's inside!